src/test/java/marquez/db/dao/TestOwnerDAO.java

```diff
@@ -0,0 +1,21 @@
+package marquez.db.dao;
+
+import org.junit.ClassRule;
+import org.junit.Test;
+
+import marquez.api.Owner;
+import marquez.db.dao.fixtures.DAOSetup;
+
+public class TestOwnerDAO {
+
+	@ClassRule
+	public static final DAOSetup daoSetup = new DAOSetup();
```

Comment:
I'm wondering if we can simplify the app setup with:

```java
   @ClassRule
   static final DropwizardAppRule<MarquezConfiguration> RULE =
     new DropwizardAppRule<MarquezConfiguration>(
       MarquezApplication.class,
       ResourceHelpers.resourceFilePath("config.yaml"));

```

source: http://dropwizard.readthedocs.io/en/stable/manual/testing.html

Reply:
If we decided to use testcontainers, the rule would be defined simply as:

```java
 @Rule
public PostgreSQLContainer postgres = new PostgreSQLContainer();
```

source: [`SimplePostgreSQLTest`](https://github.com/testcontainers/testcontainers-java/blob/master/modules/jdbc-test/src/test/java/org/testcontainers/junit/SimplePostgreSQLTest.java)

Reply:
The DAOSetup class is trying to simplify further the DropwizardAppRule, since we know the Config and App classes, have a config file etc. I started with the DARule.